### PR TITLE
fix some syntax errors

### PR DESCRIPTION
**Pull request by ParadeTo**

### CODE REVIEW

File: src/mainMenuScene.js [hunk 1]
```diff
@@ -26,13 +26,7 @@ const menuActionList = [
 ];
 
 class MainMenuScene {
-    inputManager;
```
Comment (jelster) [inline src/mainMenuScene.js:29]: Is there a reason you'd like this to be changed? 

The code is using standard ECMA class [field declaration](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes#field_declarations) syntax, which is additionally used throughout the rest of the code base (along with other things like private members.

**Comment by jelster** [comment]
Hi @ParadeTo thanks for submitting this PR! 

I'm not sure that I understand the point of these changes though as they aren't syntax errors as I understand it. Are you having issues with webpack recognizing JavaScript field declarations? Please open an issue or discussion so we can troubleshoot

**Review by jelster — COMMENTED** [review]
Please see my more general comment on the PR regarding troubleshooting this, thanks!

**Comment by ParadeTo** [comment]
image
I got this error when run `npm start`

**Comment by ParadeTo** [comment]
Perhaps we can add `babel-loader` to webpack?

**Comment by jelster** [comment]
This is an error that you should not expect to get when everything is setup properly, so I don't think `babel-loader` would be appropriate to add here either. 

Here are a couple things you might do to troubleshoot:

- make sure that you've updated to the latest version of webpack, not neglecting to include any globally-installed versions of web pack
- double-check your `webpack.common.js` and `webpack.dev.js` configuration
- you are running at least the latest LTS versions of Node/NPM

**Comment by AlexanderPolenov** [comment]
I have the same error on the start at chapter 5.

**Comment by jelster** [comment]
I'm going to close this PR, please make sure that you've upgraded and are using the latest version of WebPack (5+) as support for these language features was added to the main webpack tooling as part of the change from v4 to v5. Make sure to run an `npm ci` after any upgrades of node or NPM. 

If you're still having this problem after following the steps above, please open an issue and post the results of running `npm ls` in the body of the issue - after checking out the branch fresh and doing `npm ci`, here is what I saw:

```
space-truckers@0.0.1 C:\Users\joshe\source\space-truckers
+-- @babylonjs/core@5.0.0-alpha.10
+-- @babylonjs/gui@5.0.0-alpha.10
+-- @babylonjs/inspector@5.0.0-alpha.10
+-- @babylonjs/loaders@5.0.0-alpha.10
+-- @babylonjs/materials@5.0.0-alpha.10
+-- @babylonjs/post-processes@5.0.0-alpha.10
+-- @babylonjs/procedural-textures@5.0.0-alpha.10
+-- @babylonjs/serializers@5.0.0-alpha.10
+-- clean-webpack-plugin@3.0.0
+-- eslint@7.19.0
+-- file-loader@6.2.0
+-- html-webpack-plugin@4.5.1
+-- source-map-loader@2.0.0
+-- url-loader@4.1.1
+-- webpack-cli@4.5.0
+-- webpack-dev-server@3.11.2
+-- webpack-merge@5.7.3
`-- webpack@5.21.2
```

**Comment by ParadeTo** [comment]
But I got this error when run `npm ci`
image

My Node.js version is `v14.20.0`

As there is not only me who encounters the same problem. I think adding babel to compile will reduce many issues feedback.

**Comment by AlexanderPolenov** [comment]
@ParadeTo , I did `npm ci`, then `npm audit fix`. It changed my package.json. After that it started to work.
As an additional step, after you did commands above, you can try to delete node_modules folder and do `npm install` again.
Also, use the latest NodeJS version. I installed v19.something.

**Comment by ParadeTo** [comment]
@AlexanderPolenov Thank you very much, it works now!

**Pull request closed without merging**